vignettes/changes.Rmd

```diff
@@ -67,9 +71,8 @@ at onset of T1D (after age 40) and T2D (before age 40), respectively.
 
 | Version | Diabetes type | Sensitivity | Specificity | PPV   | NPV   |
 |---------|---------------|-------------|-------------|-------|-------|
-| 1.1     | T1D           | 0.781       | 0.999       | 0.949 | 0.997 |
-| 1.1     | T2D           | 0.943       | 0.989       | 0.879 | 0.995 |
-
+| 1.2     | T1D           | 0.788       | 0.999       | 0.947 | 0.997 |
+| 1.2     | T2D           | 0.940       | 0.990       | 0.881 | 0.995 |
 ## Potential future changes
```

Comment:
```suggestion

## Potential future changes
```

Reply:
I agree, this took way longer than anticipated. Not worth the time at the current stage.